internal/testutils/provider.go

```diff
@@ -225,6 +228,22 @@ type MockProviderInfoer struct {
 	GroupsErr bool
 }
 
+// CheckTokenScopes checks if the token has the required scopes.
+func (p *MockProviderInfoer) CheckTokenScopes(token *oauth2.Token) {
+	scopesStr, ok := token.Extra("scope").(string)
+	if !ok {
+		panic(fmt.Sprintf("Failed to cast token scopes to string: %v", token.Extra("scope")))
+	}
+	fmt.Println("Scopes: ", scopesStr)
```

Comment:
I think this is leftover of some debugging session, no?

Reply:
Oops, right